src/constants/constants.ts

```diff
@@ -91,7 +91,7 @@ export const ROUTES = {
   HOME: '/home',
   DATA_REGULATION: '/data-regulation',
   LOGOUT: '/logout',
-  HUB: '/hub',
+  HUB: '/hub?SkipWelcomeScreen=true',
```

Comment:
All links within the app should go to /hub?SkipWelcomeScreen=true if going to the Hub.  The mobile app will still point to the plain '/hub' and may end up showing the Welcome Screen.